packages/bpk-component-nudger/README.md

```diff
@@ -47,19 +49,116 @@ class App extends Component {
-| value                 | number                        | true     | -             |
-| className             | string                        | false    | null          |
-| buttonType            | oneOf('secondary', 'outline') | false    | secondary     |
+### BpkNudger
+
+| Property            | PropType                      | Required | Default Value |
+| ------------------- | ----------------------------- | -------- | ------------- |
+| id                  | string                        | true     | -             |
+| decreaseButtonLabel | string                        | true     | -             |
+| increaseButtonLabel | string                        | true     | -             |
+| max                 | number                        | true     | -             |
+| min                 | number                        | true     | -             |
+| onChange            | func                          | true     | -             |
+| value               | number                        | true     | -             |
+| className           | string                        | false    | null          |
+| buttonType          | oneOf('secondary', 'outline') | false    | secondary     |
+
+### BpkConfigurableNudger
+
+| Property            | PropType                      | Required | Default Value |
+| ------------------- | ----------------------------- | -------- | ------------- |
+| id                  | string                        | true     | -             |
+| decreaseButtonLabel | string                        | true     | -             |
+| increaseButtonLabel | string                        | true     | -             |
+| max                 | number                        | true     | -             |
+| min                 | number                        | true     | -             |
+| onChange            | func                          | true     | -             |
+| value               | number                        | true     | -             |
+| compareValues       | func                          | true     | -             |
+| incrementValue      | func                          | true     | -             |
+| decrementValue      | func                          | true     | -             |
+| formatValue         | func                          | true     | -             |
+| className           | string                        | false    | null          |
+| inputClassName      | string                        | false    | null          |
+| buttonType          | oneOf('secondary', 'outline') | false    | secondary     |
+
+### `compareValues`
+
+A comparator function similar to [Array.prototype.sort()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Description) for comparing two values in your data set.
+
+The function is used to take either the min or max value provided and compare it to the current value to determine if the value can be incremented or decremented.
```

Comment:
Can we mention explicitly when people should return a value less than 0, equal to zero, and greater than zero here? Saves people going to look at `Array.prototype.sort`.